app/models/manageiq/providers/vmware/infra_manager/storage.rb

```diff
@@ -1,3 +1,5 @@
 class ManageIQ::Providers::Vmware::InfraManager::Storage < ManageIQ::Providers::InfraManager::Storage
   include ManageIQ::Providers::Vmware::InfraManager::EmsRefObjMixin
+
+  supports :smartstate_analysis
```

Comment:
@nasark do we need to also check `storage.nil?` and https://github.com/ManageIQ/manageiq-providers-ovirt/pull/653/files#diff-24aea1f97754a89593e8311e8ea2bf2d897a159336d1f1c578d17fbb1964506dR10  here?